public/style.css

```diff
@@ -1,31 +1,494 @@
 body {
-  font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```

Comment:
i cant code review css LOL too much
basic rule of thumb- try to write as little css as possible while achieving as much as possible